example/macos/ExampleWindow.swift

```diff
@@ -18,9 +18,12 @@ class ExampleWindow: NSWindow {
   @IBOutlet weak var flutterViewController: FLEViewController!
 
   override func awakeFromNib() {
-    flutterViewController.add(FLEColorPanelPlugin())
-    flutterViewController.add(FLEFileChooserPlugin())
-    flutterViewController.add(FLEMenubarPlugin())
+    FLEColorPanelPlugin.register(
+      with: flutterViewController.registrar(forPlugin: "FLEColorPanelPlugin"))
+    FLEFileChooserPlugin.register(
```

Comment:
the wiki seem to contain only the usage in ObjectiveC, this I guess this code shows correctly how to use the file chooser plugin for instance

Reply:
> the wiki seem to contain only the usage in ObjectiveC

Calling plugin APIs from Swift is exactly the same as calling any other ObjC code from Swift.